std_msgs/Header header
int32 target_x
int32 target_y
float32 pseudo_distance
int32 image_h
int32 image_w
